omnibus/config/software/libarchive.rb

```diff
@@ -0,0 +1,8 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+name 'libarchive'
+default_version '3.5.2'
+license_file "https://raw.githubusercontent.com/libarchive/libarchive/master/COPYING"
```

Comment:
The `license` DSL should also be specified, to give a name to the license (otherwise Omnibus outputs a warning because of an `Unspecified` license, and the `LICENSE` file shipped in the package will reflect that). Since that license is unique to `libarchive`, you can just add:
```suggestion
license "Libarchive"
license_file "https://raw.githubusercontent.com/libarchive/libarchive/master/COPYING"
```